meetings/2022-05-02.md

```diff
@@ -0,0 +1,118 @@
+* AvdB: Maybe penny isn't a best example but inrupt pod browser seems widely used by the community. Maybe post an issue on the Inrupt PodBrowser GitHub repository.
+* AC: Pod browser has very recently started to support client id documents. There are still cases where it doesn't work. For me the question is what can the AuthN panel do to improve the situation with implementations. We can have good patterns documenting how authentication flows work. This can include improving the primer. We could also provide better information about _why_ it is useful and explain why it is preffered over DCR. Nicolas is very involved with client tooling. One of the issues is that there are so many permutations of what servers support, and clients need to be able to navigate those permutations.
+* AvdB: Using the PB example since that was a common starting point. Currently, Client ID documents are a SHOULD not a MUST (same with DCR). There may be reasons not to force clients to use client id documents, how can we better encourage people to use these.
+* AvdB: let me know if there is anything I can do to help with this
+* AC: There are so many permutations since there are so many contexts where clients try to authenticate. Give all of them we can't always say that client MUST do it this way. We are also trying to move ecosystemm along in a step by step fashion. If we don't allow other mechanisms than we get into compatibility issues
+* eP: I'd like to consider making Client Identifiers required for an OP, otherwise it's not Solid compliant. That would also give more confidence by the client.
+* AC: It would be helpful to add paragraph that states that OP MUST support Client ID documents.
+* ...: There may be ambiguity with regards what is and what isn't dereferencable
+
+ACTION: AC to clarify requirements for OP support of Client ID documents
+
+* AvdB: is there a case where the `client_id` value is dereferencable but _not_ a Client Id Document?
+* AC: I think there are 2 parts. 1. If it is dereferencable, if it is 2. Fetch the document and check if it is `application/ld+json`, it must be valid JSON, it also needs to use the normative `@context`.
+* ...: Let's say we have some other specification which also defines dereferencable client identifiers, not JSON-LD. When OP tries to dereference it, it would need to have condition based on the format of the client id document. Even if both are JSON-LD it could check for the `@context`.
```

Comment:
```suggestion
* ...: Let's say we have some other specification which also defines dereferenceable client identifiers, which do not deliver JSON-LD. When OP tries to dereference it, it would need to have conditions based on the format of the client ID document. Even if both are JSON-LD, it could check for the `@context`.
```